spec/helper.rb

```diff
@@ -18,7 +18,9 @@
   SimpleCov::Formatter::HTMLFormatter
 ]
 
-SimpleCov.start
+SimpleCov.start do
+  add_filter 'spec/support'
```

Comment:
A proposal to avoid some code that will never be covered if there is no test failures.